cves/kernel/CVE-2016-4998.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: underprivileged users were able to call a command normally limited to root. This allows for underprivileged user root access.
```

Comment:
First sentence should be capitalized. Also the official CVE description contains more information about the vulnerability than this description does. As a reader, I'm left wondering "What was the command? How were underprivileged users able to access it? What impacts could exploiting this vulnerability have?"